orchestra/static/dist/main.js

```diff
@@ -69232,6 +69233,13 @@ function teamInfoCard(orchestraApi) {
         });
       };
 
+      teamInfoCard.restaff = function (taskId, stepSlug) {
+        teamInfoCard.sentStaffBotRequest[stepSlug] = 'Sending request...';
```

Comment:
Seems like this logic is similar to https://github.com/b12io/orchestra/blob/6b2d3b652c69f13cf4431da786c0d0d8156bf98b/orchestra/static/orchestra/project-management/assignments-vis.es6.js#L185-L195

I am wondering we refactor so that both places? Also, should we add a exception handling logic with error message?

Reply:
This code changes the DOM model, but in my case we have a html template.
I forgot to add error handler and I'll add it soon 👍 Thanks for catching this.